Marsiple/Common/Network/PostsFetcher.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  PostsFetcher.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 17/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import Foundation
+
+struct PostsFetcher {
+    func get(succes: @escaping ([Post]) -> Void, faliure: @escaping (String) -> Void) {
```

Comment:
We could use the `LocalizedError` protocol to model our error messages, instead of passing `String` instances directly via our `failure` callbacks. 

That way we can strongly-type or error system.